client/src/containers/KsqlDB/KsqlDBList/KsqlDBInfo/KsqlDBInfo.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
```

Comment:
Not sure how to comment on the package lock file line... (sorry)

In the current dev branch we added a nmvrc so you can use [nvm](https://github.com/nvm-sh/nvm) to select the "right" node/npm version. 

This should get rid of the file change all together (or lead to a much smaller change)

Reply:
should be the same as I haven't changed anything. I updated my dev branch.

Reply:
maybe something in the way we run the app in dev? I'll investigate.